recipe/gen_patch_json.py

```diff
@@ -800,6 +800,11 @@ def _gen_new_index(repodata, subdir):
                     record["track_features"] = _extract_track_feature(
                         record, feat)
 
+        # add track_features to old python_abi pypy packages
+        if (record_name == 'python_abi' and 'pypy' in record['build'] and
+                "track_features" not in record):
+            record["track_features"] = "pypy"
```

Comment:
Do we want to version this feature with the pypy version?

Reply:
No, we want to match this with the `track_features` in python.